proto/sentry_protos/snuba/v1/endpoint_trace_item_table.proto

```diff
@@ -21,6 +21,7 @@ message TraceItemTableRequest {
   uint32 limit = 6;
   PageToken page_token = 7;
   repeated VirtualColumnContext virtual_column_contexts = 8;
+  bool debug = 10;
```

Comment:
Debug flag will be used to add QueryStats and QueryMetadata in the response

Reply:
This should go in the `RequestMeta` object so it can be generally available.